src/client/pages/Checkout/CheckoutPayment/ContactInformation/ContactInformation.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { Headline } from 'components/Headline/Headline'
+import { InputField } from 'components/inputs'
+import { useTextKeys } from 'utils/textKeys'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { useFeature, Features } from 'utils/hooks/useFeature'
+import { MEDIUM_SMALL_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { Divider } from '../../shared/Divider'
+
+type ContactInfoData = {
+  data: {
+    firstName?: string
+    lastName?: string
+    ssn?: string
+    email?: string
+  }
+}
+
+const Wrapper = styled.div`
+  ${MEDIUM_SMALL_SCREEN_MEDIA_QUERY} {
+    width: 100%;
+    max-width: 628px;
+  }
+`
+
+const InputFieldsWrapper = styled.div`
+  display: grid;
+  width: 100%;
```

Comment:
Nit
I think you can remove `width: 100%` because `grid` containers are block level elements and therefore they take all available horizontal space by default.